lib/kafka/broker.rb

```diff
@@ -30,6 +30,20 @@ def fetch_metadata(**options)
       @connection.send_request(request, response_class)
     end
 
+    def fetch_messages(**options)
```

Comment:
This API is still in flux, and Broker is an internal concept. I'm thinking about ways to get rid of it completely. I'll exhaustively document all user-facing APIs once they're stable.